Formula/alp.rb

```diff
@@ -0,0 +1,28 @@
+class Alp < Formula
+  desc "Access Log Profiler"
+  homepage "https://github.com/tkuchiki/alp"
+  url "https://github.com/tkuchiki/alp/archive/v1.0.1.tar.gz"
+  sha256 "7206f60efcca9a4f149d6abdcaf806f290fcf16237b1f7528e40628c37816449"
+  head "https://github.com/tkuchiki/alp.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
```

Comment:
Is there any other formulae setting this variable like this?

Reply:
`akamai`, `jabba`, `deis`, `peco`, and [a number of others](https://github.com/Homebrew/homebrew-core/search?q=GLIDE_HOME&unscoped_q=GLIDE_HOME) set `GLIDE_HOME`.

Reply:
This variable is unnecessary for alp and I removed it.